app/components/settingsAuthentication/settingsAuthenticationController.js

```diff
@@ -74,6 +76,13 @@ function ($q, $scope, Notifications, SettingsService, FileUploadService) {
     });
   };
 
+  // Add default port if :port is not defined in URL
+  function addLDAPDefaultPort(settings, tlsEnabled) {
+    if (settings.LDAPSettings.URL.split(':').length === 1) {
```

Comment:
Could you use `indexOf` instead of `split` here to check for the presence of the `:`  char? 

See https://www.w3schools.com/jsref/jsref_indexof.asp

Reply:
Sure! Just ported the golang code to JS, but is better to use indexOf, you're right. Give me a minute.